sklearn/ensemble/tests/test_voting.py

```diff
@@ -98,7 +123,9 @@ def test_majority_label_iris():
 
 @pytest.mark.filterwarnings('ignore:The default value of n_estimators')
 def test_tie_situation():
-    """Check voting classifier selects smaller class label in tie situation."""
+    """
```

Comment:
Please do not change unrelated things. It makes your contribution harder to review and may introduce merge conflicts to other pull requests.